notebooks/MosvizExample.ipynb

```diff
@@ -262,12 +262,39 @@
     "mosviz_no_images.load_metadata(images)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Alternate ways to show the Mosviz UI\n",
+    "\n",
+    "The above shows the mosviz UI inline with the notebook.  If you are in jupyter lab you can instead have the mosviz UI appear in a separate tab or as a \"sidecar\" (a split UI pane to the side of the notebook).  The below should do this (but only in jupyterlab.  If you are in the \"classic\" notebook it will show the same thing as `mosviz.app`)"
```

Comment:
```suggestion
    "The above shows the Mosviz UI inline with the notebook.  If you are in JupyterLab you can instead have the Mosviz UI appear in a separate tab or as a \"sidecar\" (a split UI pane to the side of the notebook).  The below should do this (but only in JupyterLab.  If you are in the \"classic\" notebook it will show the same thing as `mosviz.app`)"
```